### PR TITLE
Added more env vars to be configurable, all of them are required to run the server.

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
### Related issues
- https://github.com/threefoldtech/tfgrid_dashboard/issues/484

### CODE REVIEW

File: scripts/build-env.sh [hunk 1]
```diff
@@ -1,4 +1,8 @@
-#!/bin/sh
+#!/bin/bash
+
+REQUIRED_ENV_VARS=( GRAPHQL_URL GRIDPROXY_URL VERSION API_URL ACTIVATION_SERVICE_URL PLAYGROUND_URL RELAY )
```
Comment (AhmedHanafy725) [inline scripts/build-env.sh]: i think API_URL is confusing, let's name it substrate instead
```suggestion
REQUIRED_ENV_VARS=( GRAPHQL_URL GRIDPROXY_URL VERSION SUBSTRATE_URL ACTIVATION_SERVICE_URL PLAYGROUND_URL RELAY_URL )
```

File: scripts/build-env.sh [hunk 2]
```diff
@@ -7,100 +11,70 @@ else
...
     BRIDGE_TFT_ADDRESS=GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG
-    ACTIVATION_SERVICE_URL="https://activation.dev.grid.tf"
     EXPLORER_URL="https://explorer.dev.grid.tf/"
     STELLAR_HORIZON_URL="https://horizon-testnet.stellar.org"
-    PLAYGROUND_URL="https://play.dev.grid.tf"
-    RELAY="relay.dev.grid.tf"
+    ;;
+  *"qa"*)
+    TFT_ASSET_ISSUER=GA47YZA3PKFUZMPLQ3B5F2E3CJIB57TGGU7SPCQT2WAEYKN766PWIMB3
+    BRIDGE_TFT_ADDRESS=GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG
```
Comment (AhmedHanafy725) [inline scripts/build-env.sh]: I think the bridge address should be configurable as well

File: scripts/build-env.sh [hunk 3]
```diff
@@ -7,100 +11,70 @@ else
...
+    EXPLORER_URL="https://explorer.qa.grid.tf/"
```
Comment (AhmedHanafy725) [inline scripts/build-env.sh]: do we use the explorer url? that's already gone

File: scripts/build-env.sh [hunk 4]
```diff
@@ -7,100 +11,70 @@ else
...
 GAS_PRICE="80TFT"
 CHAIN_ID="threefold-hub-testnet"
 
-case $GQL_URL in
-  *"dev"*)
-    NETWORK="dev"
-    GQL_URL="https://graphql.dev.grid.tf/graphql"
-	  PROXY_URL="https://gridproxy.dev.grid.tf"
-    POLKADOT_URL="wss://tfchain.dev.grid.tf/ws"
+case $NETWORK in
```
Comment (AhmedHanafy725) [inline scripts/build-env.sh]: the network env now has no sense to be `dev`, `qa`, `test`, and `main`, it's used only for stellar tft asset and horizon url, so I think it should be called `STELLAR_NETWORK` and it should be `test`, `std` or `main` only

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: scripts/build-env.sh (hunk 1 above)
Comment (Mahmoud-Emad) [reply scripts/build-env.sh:3]: Done

File: scripts/build-env.sh (hunk 4 above)
Comment (Mahmoud-Emad) [reply scripts/build-env.sh:33]: We still need the tf-chain network it reflects `explorer` btw, how do you point to the target environment?

Comment (mahmoud-emad) [reply scripts/build-env.sh]: We still need the tf-chain network it reflects explorer btw, how do you point to the target environment?

File: scripts/build-env.sh (hunk 1 above)
Comment (mahmoud-emad) [reply scripts/build-env.sh]: Done

File: scripts/build-env.sh (hunk 2 above)
Comment (mahmoud-emad) [reply scripts/build-env.sh]: `BRIDGE_TFT_ADDRESS` and `TFT_ASSET_ISSUER`?

File: scripts/build-env.sh (hunk 3 above)
Comment (mahmoud-emad) [reply scripts/build-env.sh]: So if we do not need it I think we can replace the env NETWORK with STELLAR_NETWORK as you mentioned.

File: scripts/build-env.sh (hunk 2 above)
Comment (AhmedHanafy725) [reply scripts/build-env.sh]: `TFT_ASSET_ISSUER` depends on the stellar network, not on the tfchain

File: scripts/build-env.sh (hunk 4 above)
Comment (AhmedHanafy725) [reply scripts/build-env.sh]: using substrate url, no?

**Comment by AhmedHanafy725** [comment]
also, make sure to use the relay is not hard coded, should be used from the config

**Review by Omarabdul3ziz — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**